pkg/personname/info.go

```diff
@@ -0,0 +1,202 @@
+/*
+The personname package provides methods and data types for inspecting Person Name (PN)
+Value Representations, as defined here:
+
+http://dicom.nema.org/dicom/2013/output/chtml/part05/sect_6.2.html
+*/
+package personname
+
+import (
+	"fmt"
+	"strings"
+)
+
+const groupSep = "="
+
+// pnGroup is an enum value for the PN group types (Alphabetic, Ideographic & Phonetic).
+type pnGroup int
+
+// String representation -- mostly for formatting error messages.
+func (group pnGroup) String() string {
+	switch group {
+	case 0:
```

Comment:
order should not matter in Go, so I think you can refer to the enum consts directly here like pnGroupAlphabetic, etc. You can also declare the consts before declaring the String() method for readability if you desire.